packages/react-ui-core/src/DropDown/DefaultAnchor.js

```diff
@@ -0,0 +1,40 @@
+import React, { PureComponent } from 'react'
+import PropTypes from 'prop-types'
+import { Button } from '../Button'
+
+export default class DropDownAnchorButton extends PureComponent {
+  static propTypes = {
+    toggleVisibilty: PropTypes.func,
+    handleDocumentClick: PropTypes.func,
+    text: PropTypes.node,
+    className: PropTypes.string,
+  }
+
+  constructor(props) {
+    super(props)
+    this.handleClick = this.handleClick.bind(this)
+  }
+
+  componentDidMount() {
```

Comment:
Do we need this addEventListener here and on its parent?

Reply:
Removed from parent. Didn't mean to leave it there.